src/classes/CRLP_Batch_Base_Skew.cls

```diff
@@ -477,13 +481,17 @@ public abstract class CRLP_Batch_Base_Skew extends CRLP_Batch_Base {
         String query = CRLP_Query_SEL.buildObjectQueryForRollup(summaryObjectType) + ' WHERE Id IN :parentIds';
         List<SObject> parents = Database.query(query);
 
+        CRLP_RollupProcessingOptions.ProcessingOptions options = new CRLP_RollupProcessingOptions.ProcessingOptions();
+        options.useRollupDefsByParentIdMap = true;
```

Comment:
From the context of this class or `CRLP_ApiExecuteRollups`, there is not so much meaning given by setting `useRollupDefsByParentIdMap` to true. Is this basically `isSkewProcessingMode` or simply `isSkewMode`?

Reply:
Good question. It was originally tied strictly to SkewMode processing, however rollups through the Api is sort of a blend of modes. It's SkewMode mostly, but not 100%. So, to separate the concepts I created this new property in the ProcessingOptions class. It's always True in Skew Mode, but it's not always False in non-Skew mode.